metadata.json

```diff
@@ -1,8 +1,24 @@
 {
   "TestModule1": {
-    "fields": []
+    "fields": {
+      "testField3": {
```

Comment:
`testField3` and `testField4` are just a couple of examples of fields that might have falsy values.
`testField5` on the other hand is just here to make sure that it actually gets automagically filled when nothing is supplied.